OpenTabletDriver.Console/Program.cs

```diff
@@ -20,6 +20,11 @@ public static async Task Main(string[] args)
             }
 
             await Driver.Connect();
+            Driver.Disconnected += (sender, eventArgs) =>
+            {
+                System.Console.Out.Write("Daemon disconnected, exiting...");
```

Comment:
This needs a newline or it can make the terminal wonky.

```suggestion
                System.Console.Out.WriteLine("Daemon disconnected, exiting...");
```

Reply:
Oops, nice catch